Week1/homework/app.js

```diff
@@ -1,11 +1,169 @@
 'use strict';
 
+// 1-1 Open the apps.js and start by declaring an array that contains 10 strings
```

Comment:
We need not have this comment as it doesn't really add much value here. It would have been helpful to you during development but we need not have it in the final version, right? 😉

Reply:
I deleted unnecessarily comment!